pkg/kapp/clusterapply/change_set_view.go

```diff
@@ -45,6 +48,28 @@ func (v *ChangeSetView) Print(ui ui.UI) {
 	}
 }
 
+const (
+	// 0.5 MB in bytes
+	diffChangesStringMaxSize = 524288
+)
+
+func (v *ChangeSetView) DiffChangesString() string {
```

Comment:
instead of defining a new method could users be using Print with a different UI (some kind of in memory ui). there is also a question of how much context to include in the diff. today i guess it's defaulted to 1 line of context i think. summary section might also be useful (but may be always turned on)?

Reply:
It does make sense to store the summary at all times. 
You man to say that the Print statements will get buffered to this memoized UI, from where it can be flushed to the config map? I believe that will make including different parts of the output (like summary) easier.
I will explore that 🤔